autode/opt/optimisers/base.py

```diff
@@ -886,90 +831,314 @@ def print_geometries(self, filename: Optional[str] = None) -> None:
             else str(filename)
         )
 
-        self._history.print_geometries(self._species, filename=filename)
+        OptimiserHistory.print_geometries(
+            self._history, self._species, filename=filename
+        )
```

Comment:
if `self._history` is a `OptimiserHistory` then this might be cleaner an instance method
```python
         self._history,.print_geometries(
            self._species, filename=filename
        )
```